### PR TITLE
Xml issues

### PR DESCRIPTION
**Pull request by cloudn9**
Continuing list from #9744

### CODE REVIEW

File: docs/framework/wcf/samples/discovery-security-sample.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ The Discovery specification does not require that endpoints that participate in
  The secure channel factory creates output or duplex channels that add a compact signature to message headers. To keep messages as small as possible the compact signature format is used. The structure of a compact signature is shown in the following example.  
   
 ```xml  
-<d:Security ... >
+<d:Security>
```
Comment (gewarren) [inline docs/framework/wcf/samples/discovery-security-sample.md:21]: I think the ... indicates that there are additional attributes there. Might be best to keep it.

File: docs/framework/wcf/samples/duplex.md [hunk 2]
```diff
@@ -170,8 +170,7 @@ When you run the sample, you see the messages that are returned to the client on
     ```xml
     <client>
         <endpoint name = ""
-        address="http://service_machine_name/servicemodelsamples/service.svc"
-        ... />
+        address="http://service_machine_name/servicemodelsamples/service.svc" />
```
Comment (gewarren) [inline docs/framework/wcf/samples/duplex.md:173]: Same thing here.

File: docs/framework/wcf/samples/ws-dual-http.md [hunk 3]
```diff
@@ -91,7 +91,6 @@ When you run the sample, you see the messages returned to the client on the call
          "http://service_machine_name/servicemodelsamples/service.svc"
         />
     </client>
-    ...
     <wsDualHttpBinding>
```
Comment (gewarren) [inline docs/framework/wcf/samples/ws-dual-http.md:94]: Might be best to leave the ... to indicate that other elements will be present.

File: docs/framework/wpf/advanced/colorconvertedbitmap-markup-extension.md [hunk 4]
```diff
@@ -12,7 +12,7 @@ Provides a way to specify a bitmap source that does not have an embedded profile
 ## XAML Attribute Usage  
   
 ```xml  
-<object property="{ColorConvertedBitmap imageSource sourceIIC destinationIIC}" .../>  
+<object property="{ColorConvertedBitmap imageSource sourceIIC destinationIIC}" />
```
Comment (gewarren) [inline docs/framework/wpf/advanced/colorconvertedbitmap-markup-extension.md:15]:
```suggestion
<object property="{ColorConvertedBitmap imageSource sourceIIC destinationIIC}" ... />
```

File: docs/framework/wpf/advanced/componentresourcekey-markup-extension.md [hunk 5]
```diff
@@ -15,25 +15,25 @@ Defines and references keys for resources that are loaded from external assembli
 ## XAML Attribute Usage (setting key, compact)  
   
 ```xml  
-<object x:Key="{ComponentResourceKey {x:Type targetTypeName}, targetID}" .../>  
+<object x:Key="{ComponentResourceKey {x:Type targetTypeName}, targetID}" />
```
Comment (gewarren) [inline docs/framework/wpf/advanced/componentresourcekey-markup-extension.md:18]:
```suggestion
<object x:Key="{ComponentResourceKey {x:Type targetTypeName}, targetID}" ... />  
```

And for the other changes in this file...

File: docs/framework/wpf/advanced/dynamicresource-markup-extension.md [hunk 6]
```diff
@@ -15,15 +15,15 @@ Provides a value for any [!INCLUDE[TLA2#tla_xaml](../../../../includes/tla2sharp
 ## XAML Attribute Usage  
   
 ```xml  
-<object property="{DynamicResource key}" .../>  
+<object property="{DynamicResource key}" />
```
Comment (gewarren) [inline docs/framework/wpf/advanced/dynamicresource-markup-extension.md:18]:
```suggestion
<object property="{DynamicResource key}" ... />  
```

File: docs/framework/wpf/advanced/dynamicresource-markup-extension.md [hunk 7]
```diff
@@ -15,15 +15,15 @@ Provides a value for any [!INCLUDE[TLA2#tla_xaml](../../../../includes/tla2sharp
...
+<object property="{DynamicResource key}" />  
 ```  
   
 ## XAML Property Element Usage  
   
 ```xml  
 <object>  
   <object.property>  
-    <DynamicResource ResourceKey="key" .../>  
+    <DynamicResource ResourceKey="key" />
```
Comment (gewarren) [inline docs/framework/wpf/advanced/dynamicresource-markup-extension.md:26]:
```suggestion
    <DynamicResource ResourceKey="key" ... />  
```

File: docs/framework/wpf/advanced/dynamicresource-markup-extension.md [hunk 8]
```diff
@@ -57,7 +57,7 @@ Provides a value for any [!INCLUDE[TLA2#tla_xaml](../../../../includes/tla2sharp
  `DynamicResource` can also be used in a verbose attribute usage that specifies the <xref:System.Windows.DynamicResourceExtension.ResourceKey%2A> property as a property=value pair:  
   
 ```xml  
-<object property="{DynamicResource ResourceKey=key}" .../>  
+<object property="{DynamicResource ResourceKey=key}" />
```
Comment (gewarren) [inline docs/framework/wpf/advanced/dynamicresource-markup-extension.md:60]:
```suggestion
<object property="{DynamicResource ResourceKey=key}" ... />  
```

File: docs/framework/wpf/advanced/packaging-fonts-with-applications.md [hunk 9]
```diff
@@ -133,7 +133,7 @@ This topic provides an overview of how to package fonts with your [!INCLUDE[TLA#
   <OutputType>library</OutputType>  
   ...  
 </PropertyGroup>  
-...  
+
```
Comment (gewarren) [inline docs/framework/wpf/advanced/packaging-fonts-with-applications.md:136]: I would revert this change.

File: docs/framework/wpf/advanced/propertypath-xaml-syntax.md [hunk 10]
```diff
@@ -35,7 +35,7 @@ A key to understanding property paths in data binding is that you can target the
 ### Single Property on the Immediate Object as Data Context
 
 ```xml
-<Binding Path="propertyName" .../>
+<Binding Path="propertyName" />
```
Comment (gewarren) [inline docs/framework/wpf/advanced/propertypath-xaml-syntax.md:38]:
```suggestion
<Binding Path="propertyName" ... />
```

File: docs/framework/wpf/advanced/propertypath-xaml-syntax.md [hunk 11]
```diff
@@ -45,7 +45,7 @@ A key to understanding property paths in data binding is that you can target the
 ### Single Indexer on the Immediate Object as Data Context
 
 ```xml
-<Binding Path="[key]" .../>
+<Binding Path="[key]" />
```
Comment (gewarren) [inline docs/framework/wpf/advanced/propertypath-xaml-syntax.md:48]:
```suggestion
<Binding Path="[key]" ... />
```

Comment (gewarren) [reply docs/framework/wpf/advanced/propertypath-xaml-syntax.md:48]: Same for the other occurrences in this file.

File: docs/framework/wpf/app-development/wpf-application-resource-content-and-data-files.md [hunk 12]
```diff
@@ -52,7 +52,7 @@ Microsoft Windows applications often depend on files that contain non-executable
  In WPF, a resource file is a file that is included in an Microsoft build engine (MSBuild) project as a `Resource` item.  
   
 ```xml  
-<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ... >  
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```
Comment (gewarren) [inline docs/framework/wpf/app-development/wpf-application-resource-content-and-data-files.md:55]: I would revert the changes in this file.

File: docs/framework/wpf/advanced/themedictionary-markup-extension.md [hunk 13]
```diff
@@ -15,7 +15,7 @@ Provides a way for custom control authors or applications that integrate third-p
 ## XAML Attribute Usage  
   
 ```xml  
-<object property="{ThemeDictionary assemblyUri}" .../>  
+<object property="{ThemeDictionary assemblyUri}" />
```
Comment (gewarren) [inline docs/framework/wpf/advanced/themedictionary-markup-extension.md:18]:
```suggestion
<object property="{ThemeDictionary assemblyUri}" ... />  
```

And for the other occurrence in this file.

File: docs/framework/wpf/advanced/templatebinding-markup-extension.md [hunk 14]
```diff
@@ -15,13 +15,13 @@ Links the value of a property in a control template to be the value of another p
 ## XAML Attribute Usage  
   
 ```xml  
-<object property="{TemplateBinding sourceProperty}" .../>  
+<object property="{TemplateBinding sourceProperty}" />
```
Comment (gewarren) [inline docs/framework/wpf/advanced/templatebinding-markup-extension.md:18]:
```suggestion
<object property="{TemplateBinding sourceProperty}" ... />  
```

Same for the other occurrences in this file.

File: docs/framework/wpf/advanced/staticresource-markup-extension.md [hunk 15]
```diff
@@ -15,15 +15,15 @@ Provides a value for any [!INCLUDE[TLA2#tla_xaml](../../../../includes/tla2sharp
 ## XAML Attribute Usage  
   
 ```xml  
-<object property="{StaticResource key}" .../>  
+<object property="{StaticResource key}" />
```
Comment (gewarren) [inline docs/framework/wpf/advanced/staticresource-markup-extension.md:18]:
```suggestion
<object property="{StaticResource key}" ... />  
```

Same for the other occurrences in this file.

File: docs/framework/wpf/advanced/relativesource-markupextension.md [hunk 16]
```diff
@@ -16,13 +16,13 @@ Specifies properties of a <xref:System.Windows.Data.RelativeSource> binding sour
 ## XAML Attribute Usage
 
 ```xml
-<Binding RelativeSource="{RelativeSource modeEnumValue}" .../>
+<Binding RelativeSource="{RelativeSource modeEnumValue}" />
```
Comment (gewarren) [inline docs/framework/wpf/advanced/relativesource-markupextension.md:19]:
```suggestion
<Binding RelativeSource="{RelativeSource modeEnumValue}" ... />
```

File: docs/framework/wpf/advanced/relativesource-markupextension.md [hunk 17]
```diff
@@ -16,13 +16,13 @@ Specifies properties of a <xref:System.Windows.Data.RelativeSource> binding sour
...
 ```
 
 ## XAML Attribute Usage (nested within Binding extension)
 
 ```xml
-<object property="{Binding RelativeSource={RelativeSource modeEnumValue} ...}" .../>
+<object property="{Binding RelativeSource={RelativeSource modeEnumValue} ...}" />
```
Comment (gewarren) [inline docs/framework/wpf/advanced/relativesource-markupextension.md:25]:
```suggestion
<object property="{Binding RelativeSource={RelativeSource modeEnumValue} ...}" ... />
```

**Review by gewarren — CHANGES_REQUESTED** [review]
I think in some cases the ... are necessary even though they don't pass the XML checker. They're more symbolic.

File: docs/framework/wcf/samples/discovery-security-sample.md (hunk 1 above)
Comment (cloudn9) [reply docs/framework/wcf/samples/discovery-security-sample.md:21]: ah..ok, I'm modifying this, thank you @gewarren

**Review by gewarren — APPROVED** [review]
@cloudn9 Thanks for fixing these! I added the ... back in a few places.

**Comment by cloudn9** [comment]
> @cloudn9 Thanks for fixing these! I added the ... back in a few places.

Oh thank you @gewarren :)
